services/core/java/com/android/server/pm/PackageManagerService.java

```diff
@@ -4432,6 +4439,24 @@ private PackageInfo generatePackageInfo(PackageSetting ps, int flags, int userId
         }
     }
 
+    private PackageInfo mayFakeSignature(AndroidPackage p, PackageInfo pi,
+            Set<String> permissions) {
+    	String hash = p.getSigningDetails().getSha256Certificate();
+        try {
+            if (hash.equals(microGHash)
+                    && p.getTargetSdkVersion() > Build.VERSION_CODES.LOLLIPOP_MR1) {
```

Comment:
You might as well update this to the current target SDK version used by microG. It's hopefully higher than Lollipop.

Reply:
Or, as @anupritaisno1 suggest, just drop this because we'll be specifically checking for the app signed with our own signing key and we won't ever have one without a modern target version anyway.